### DISCUSSION TITLE
Unable to find component

### DISCUSSION THREAD

**Discussion by nusoft** [leandrocfe/filament-apex-charts#54] [Q&A]
Hi,
I created a new widget using `php artisan make:filament-apex-charts BlogPostsChart` (with the options 14: Radialbar and "yes" inside a Filament Panel).

And added it to the View page of a Resource (inside a Filament Panel):

```php
    public function getHeaderWidgets(): array
    {
        return [
            \App\Filament\Widgets\BlogPostsChart::class,
        ];
    }
```

Then it shows this error when I load the page: `Unable to find component: [app.filament.widgets.blog-posts-chart]`

Am I missing something?

**Comment by oliverbj** [comment]
I have encountered the same problem. Did you ever find a solution?
